src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -138,6 +154,13 @@ public async Task<Response<ConfigurationSetting>> UpdateAsync(ConfigurationSetti
                 message.AddHeader(MediaTypeKeyValueApplicationHeader);
                 message.AddHeader(HttpHeader.Common.JsonContentType);
                 message.AddHeader(HttpHeader.Common.CreateContentLength(content.Length));
+                if(!string.IsNullOrEmpty(setting.ETag))
```

Comment:
BTW, we should change the ETag property to be of ETag type, not string.

Reply:
Yes. I have that planned in another PR